src/NServiceBus.Core.Tests/OpenTelemetry/MeterTests.cs

```diff
@@ -6,22 +6,27 @@
 using Particular.Approvals;
 
 [TestFixture]
-public class MeterTagsTests
+public class MeterTests
 {
     [Test]
-    public void Verify_MeterTags()
+    public void Verify_MeterAPI()
     {
         var meterTags = typeof(MeterTags)
             .GetFields(BindingFlags.Public | BindingFlags.Static)
             .Where(fi => fi.IsLiteral && !fi.IsInitOnly)
             .Select(x => $"{x.Name} => {x.GetRawConstantValue()}")
             .ToList();
-
+        var metrics = typeof(Metrics)
+            .GetFields(BindingFlags.Public | BindingFlags.Static)
+            .Where(fi => fi.IsLiteral && !fi.IsInitOnly)
+            .Select(x => $"{x.Name} => {x.GetRawConstantValue()}")
+            .ToList();
         Approver.Verify(new
         {
-            Note = "Changes to meter tags should result in Meters version updates",
+            Note = "Changes to metrics API should result in Meters version updates",
```

Comment:
Instead of "Meters version updates" should this be "an update to NServiceBusMeter.Version" or something?